packages/doctrine-orm-bridge/composer.json

```diff
@@ -24,14 +24,30 @@
             "homepage": "http://php-and-symfony.matthiasnoback.nl"
         }
     ],
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/ruudk/doctrine-orm-test-service-provider.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/ruudk/doctrine-dbal-test-service-provider.git"
+        },
+        {
+            "type": "vcs",
+            "url": "https://github.com/ruudk/phpunit-test-service-container.git"
+        }
+    ],
```

Comment:
This can be removed as soon as 
https://github.com/matthiasnoback/phpunit-test-service-container/pull/8
https://github.com/matthiasnoback/doctrine-orm-test-service-provider/pull/4
https://github.com/matthiasnoback/doctrine-dbal-test-service-provider/pull/5

are merged and tagged

Reply:
https://github.com/SimpleBus/SimpleBus/issues/91